scripts/weekly-code-review.sh

```
@@ -0,0 +1,228 @@
+  echo "weekly-code-review: cannot create $OUT_DIR" >&2
+  exit 2
+}
+MD="$OUT_DIR/report.md"
+JSON="$OUT_DIR/report.json"
+TMP="$(mktemp -d)"
+trap 'rm -rf "$TMP"' EXIT
+
+DATE_UTC="$(date -u +%Y-%m-%d)"
+SHA="$(git rev-parse --short=12 HEAD 2>/dev/null || echo 'unknown')"
+REPO_ROOT="$(cd "$(dirname "$0")/.." && pwd)"
+cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)